src/main/scala/com/codacy/analysis/cli/files/FileSystemFileCollector.scala

```diff
@@ -33,10 +33,15 @@ class FileSystemFileCollector extends FileCollector[Try] {
           .filterNot(_.startsWith(".git"))
           .to[Set]
 
-      val filters: Set[Set[Path] => Set[Path]] = Set(
-        excludeGlobal(localConfiguration) _,
-        excludePrefixes(remoteConfiguration) _,
-        excludeAutoIgnores(remoteConfiguration) _)
+      val autoIgnoresFilter: Set[Path] => Set[Path] = localConfiguration.fold({ _ =>
```

Comment:
I would make an if here to be more explicit. Something like `if (localConfiguration.isDefined){`...

Reply:
Will do it with an isLeft, cant do isDefined because localConfiguration is an Either